docs/modules/ROOT/pages/index.adoc

```diff
@@ -674,6 +675,7 @@ The `url` field for a status code MUST always be the label of the *<<_identity_a
 |`cawg.identity.cbor.invalid` | The CBOR of the *<<_identity_assertion,identity assertion>>* is not valid.
 |`cawg.identity.assertion.mismatch` | The *<<_identity_assertion,identity assertion>>* contains an assertion reference that could not be found in the _<<C2PA claim>>._
 |`cawg.identity.assertion.duplicate` | The *<<_identity_assertion,identity assertion>>* contains one or more duplicate assertion references.
+|`cawg.identity.credential_revoked` | The *<<_identity_assertion,identity assertion>>* was signed using a credential that has been revoked.
```

Comment:
First, this should be more explicit: does it mean that it was signed with a credential that was revoked at the time of signing, or revoked after the signature was created? That is, "...was signed using a revoked credential" v "... a credential that has since been revoked."
Second, credential revocation is going to be a normal event (at least for verifiable credentials). Sorry for reiterating my comment from 138... 
Credentials may be revoked as part of the lifecycle of the identity and of the credential.
For example, an individual who changes part of their identity record -- their name or address -- will likely have their old credential revoked and a new one issued to them, so that the credential reflects the current state of the source identity record. Also: some credentials may have expiry dates enforced by revocation.
Two implications of this:

- Revocation is a poor signal that credential's subject did anything wrong
- Revocation will be a common (and largely inevitable) event

Possibly the key point is that the credential was not revoked at the time the identity assertion was signed